python/pfs/instmodel/spectrum.py

```diff
@@ -396,7 +396,7 @@ def __init__(self, species=None, scale=1.0):
             self.lines = self.lines[select]
 
         # Flux scale in the file is somewhat arbitrary; convert it to something of order 1.
-        super().__init__(self.lines["wavelength"], scale*self.lines["flux"]*1.0e-2)
+        super().__init__(self.lines["wavelength"], scale*self.lines["flux"]*1.0e-3)
```

Comment:
If it's arbitrary, why this specific change?

Reply:
It keeps the exposure times reasonable.